### PR TITLE
optimize:Some words are misspelled, reading the source code is slightly affected

### PR DESCRIPTION
**Pull request by about-cloud**
…ng the reading source code.

### Ⅰ. Describe what this PR did
Changed the spelling of some words.

### CODE REVIEW

**Comment by caohdgege** [comment]
考虑到用户体验的问题，如果配置项改动的话要考虑向上兼容，这边讨论之后的建议是1.5版本的话是新增retryInterval参数，但是要兼容retryInternal(标记为Deprecated，但是代码依然可以读到这个参数)。1.6的时候再把retryInternal配置项完全删除。

**Comment by caohdgege** [comment]
还有pr的标题要改一下，要用 optimize:  开头

**Comment by slievrly** [comment]
@about-cloud Can you handle code conflicts?

**Comment by a364176773** [comment]
请把未改动代码还原。

**Comment by about-cloud** [comment]
> 请把未改动代码还原。

我检查了下，那个两个测试类，里面分别创建了几个内部类用来测试不同的场景。因为原来少许代码被封装在内部类中了，所以原来的代码会有缩进，。

**Comment by about-cloud** [comment]
> 请把未改动代码还原。

spring/src/main/java/io/seata/spring/annotation/GlobalLock.java
spring/src/main/java/io/seata/spring/annotation/GlobalTransactional.java

这两个类有改动，之所以格式代码是因为原来代码没有缩进，不符开发规范。

File: rm-datasource/src/test/java/io/seata/rm/datasource/exec/LockRetryControllerTestC.java [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+import io.seata.common.DefaultValues;
+import io.seata.config.ConfigurationChangeEvent;
+import io.seata.core.constants.ConfigurationKeys;
+import io.seata.core.context.GlobalLockConfigHolder;
+import io.seata.core.model.GlobalLockConfig;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
```
Comment (slievrly) [inline rm-datasource/src/test/java/io/seata/rm/datasource/exec/LockRetryControllerTestC.java:27]: forbid to use *

File: rm-datasource/src/test/java/io/seata/rm/GlobalLockTemplateTestB.java [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+package io.seata.rm;
+
+import io.seata.core.context.GlobalLockConfigHolder;
+import io.seata.core.context.RootContext;
+import io.seata.core.model.GlobalLockConfig;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
```
Comment (slievrly) [inline rm-datasource/src/test/java/io/seata/rm/GlobalLockTemplateTestB.java:25]: +1

File: rm-datasource/src/test/java/io/seata/rm/GlobalLockTemplateTestC.java [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+package io.seata.rm;
+
+import io.seata.core.context.GlobalLockConfigHolder;
+import io.seata.core.context.RootContext;
+import io.seata.core.model.GlobalLockConfig;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
```
Comment (slievrly) [inline rm-datasource/src/test/java/io/seata/rm/GlobalLockTemplateTestC.java:25]: +1

File: rm-datasource/src/test/java/io/seata/rm/datasource/exec/LockRetryControllerTestB.java [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+import io.seata.common.DefaultValues;
+import io.seata.config.ConfigurationChangeEvent;
+import io.seata.core.constants.ConfigurationKeys;
+import io.seata.core.context.GlobalLockConfigHolder;
+import io.seata.core.model.GlobalLockConfig;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
```
Comment (slievrly) [inline rm-datasource/src/test/java/io/seata/rm/datasource/exec/LockRetryControllerTestB.java:27]: +1

File: spring/src/main/java/io/seata/spring/annotation/GlobalLock.java [hunk 5]
```diff
@@ -27,30 +27,48 @@
...
  * but the transaction need to ensure records to update(or select for update) is not in global transaction middle
  * stage<br/>
- *
  * use this annotation instead of GlobalTransaction in the situation mentioned above will help performance.
  *
+ * @author skyesx
+ * @author WangLiang/王良
+ * @author selfishlover
+ * @author jimin
+ * @author linkedme@qq.com
```
Comment (a364176773) [inline spring/src/main/java/io/seata/spring/annotation/GlobalLock.java:36]: you need to add @Author only if you refactor your code

File: tm/src/main/java/io/seata/tm/api/transaction/TransactionInfo.java [hunk 6]
```diff
@@ -22,6 +22,7 @@
 
 /**
  * @author guoyao
+ * @author linkedme@qq.com
```
Comment (a364176773) [inline tm/src/main/java/io/seata/tm/api/transaction/TransactionInfo.java:25]: +1

**Comment by a364176773** [comment]
请合并一下最新的 seata仓库develop分支代码

File: spring/src/main/java/io/seata/spring/annotation/GlobalLock.java [hunk 7]
```diff
@@ -27,30 +27,48 @@
...
      * you may use this to override global config of "client.rm.lock.retryInterval"
      * note: 0 or negative number will take no effect(which mean fall back to global config)
-     * @return
+     * <br/>
+     * will be removed in v1.6, please use the new {@link GlobalLock#lockRetryInterval()}
+     *
+     * @return lock retry interval
      */
+    @Deprecated
     int lockRetryInternal() default 0;
```
Comment (selfishlover) [inline spring/src/main/java/io/seata/spring/annotation/GlobalLock.java:56]: you can use spring @AliasFor to make the old lockRetryInternal and the new lockRetryInterval "merge" into one value, so as to keep compatibility and simply a lot of code(basically, translate those misspelled and no need to add more test).

**Comment by l81893521** [comment]
I will close this PR because that was not processed for a long time and waiting for you next contribution.

**Pull request closed without merging**